src/main/java/bio/terra/service/filedata/FileService.java

```diff
@@ -132,7 +132,7 @@ public String ingestBulkFile(
 
   public String ingestBulkFileArray(
       String datasetId, BulkLoadArrayRequestModel loadArray, AuthenticatedUserRequest userReq) {
-    String loadTag = loadArray.getLoadTag();
+    String loadTag = loadService.computeLoadTag(loadArray.getLoadTag());;
```

Comment:
I'm guessing the extra `;` is why the tests are failing

Reply:
whoops!